ldap.go

```diff
@@ -153,15 +153,37 @@ func addLDAPDescriptions(packet *ber.Packet) (err error) {
 func addControlDescriptions(packet *ber.Packet) {
 	packet.Description = "Controls"
 	for _, child := range packet.Children {
+		var value *ber.Packet
```

Comment:
can we do the same thing here where we declare `controlType := ""`, set it inside the cases, then switch on `controlType` instead of `child.Children[0].Value.(string)`?